R/gta data slicer.R

```diff
@@ -242,7 +242,7 @@ gta_data_slicer=function(data.path = "data/master_plus.Rdata",
             parameter.choices=rbind(parameter.choices,
                                     data.frame(parameter="Affected countries included:", choice=paste(affected.country, collapse = ", ")))
 
-          } else {
+          } else if (keep.affected == F){
```

Comment:
I don't mind it here, but think it is redundant. Especially so since you introduce "| !is.logical(keep.affected)" in line 232.

Reply:
dito in the below cases.